### ISSUE TITLE
I wanna see the loss_cls

### ISSUE THREAD

**Issue by ccedar** [open-mmlab/mmdetection#6878]
### I wanna see the loss value of my training like this.
2021-11-24 15:51:04,093 - mmdet - INFO - Epoch [1][50/144]	lr: 2.473e-04, eta: 6:23:17, time: 0.266, data_time: 0.046, memory: 3610, loss_rpn_cls: 0.5732, loss_rpn_bbox: 0.0419, loss_cls: 1.1295, acc: 89.1875, loss_bbox: 0.0206, loss: 1.7651
2021-11-24 15:51:15,535 - mmdet - INFO - Epoch [1][100/144]	lr: 4.970e-04, eta: 5:56:02, time: 0.229, data_time: 0.005, memory: 3610, loss_rpn_cls: 0.0810, loss_rpn_bbox: 0.0256, loss_cls: 0.0561, acc: 98.5762, loss_bbox: 0.0451, loss: 0.2078

### But, now I can only see the value right below. I can't see the value of loss.  I trained the same model and I don't know why I can't get the value of losses this time. Thank you in advance:)
2021-12-24 10:48:21,092 - mmdet - INFO - Epoch(val) [1][37]	bbox_mAP: 0.0000, bbox_mAP_50: 0.0000, bbox_mAP_75: 0.0000, bbox_mAP_s: -1.0000, bbox_mAP_m: -1.0000, bbox_mAP_l: 0.0000, bbox_mAP_copypaste: 0.000 0.000 0.000 -1.000 -1.000 0.000, segm_mAP: 0.0000, segm_mAP_50: 0.0000, segm_mAP_75: 0.0000, segm_mAP_s: -1.0000, segm_mAP_m: -1.0000, segm_mAP_l: 0.0000, segm_mAP_copypaste: 0.000 0.000 0.000 -1.000 -1.000 0.000

**Comment by BIGWangYuDong** [comment]
` Epoch(val)` this is the evaluation log

**Comment by ccedar** [comment]
> ` Epoch(val)` this is the evaluation log

I know but, I wanna get the value of losses

**Comment by BIGWangYuDong** [comment]
During test time?

**Comment by ccedar** [comment]
> During test time?

Nope. During the training

**Comment by BIGWangYuDong** [comment]
maybe your training data is too small. you can change the interval here: (such as 10 or others)

https://github.com/open-mmlab/mmdetection/blob/df212e7505e1004d82e6882eeb9466a2156972fb/configs/_base_/default_runtime.py#L3-L8

**Comment by ccedar** [comment]
> maybe your training data is too small. you can change the interval here: (such as 10 or others)
> 
> https://github.com/open-mmlab/mmdetection/blob/df212e7505e1004d82e6882eeb9466a2156972fb/configs/_base_/default_runtime.py#L3-L8

I changed my interval 50 to 10 and it works!!!!!!!!!! Thank you so much :D
